src/interfaces/IVaultToken.sol

```diff
@@ -18,18 +18,38 @@ interface IVaultToken {
 
     event Transfer(address caller, address indexed from, address indexed to, Currency indexed currency, uint256 amount);
 
-    /// @notice get the amount of user's surplus token in vault
-    function balanceOf(address user, Currency currency) external view returns (uint256);
-
-    /// @notice get the amount that user has authorized for spender to use
-    function allowance(address user, address spender, Currency currency) external view returns (uint256);
+    /// @notice get the amount of owner's surplus token in vault
+    /// @param owner The address you want to query the balance of
+    /// @param currency The currency you want to query the balance of
+    /// @return balance The balance of the specified address
+    function balanceOf(address owner, Currency currency) external view returns (uint256 balance);
+
+    /// @notice get the amount that owner has authorized for spender to use
+    /// @param owner The address of the owner
+    /// @param spender The address who is allowed to spend the owner's token
+    /// @param currency The currency the spender is allowed to spend
+    /// @return amount The amount of token the spender is allowed to spend
+    function allowance(address owner, address spender, Currency currency) external view returns (uint256);
```

Comment:
Just curious , we added the return name above,  why those interfaces did not add return name ?

Reply:
Agree, I think this one should add return name as well